log-analyzer/features/org.wso2.analytics.common.la.apim.feature/src/main/resources/gadgets/LogErrorFilteredMessage/js/main.js

```diff
@@ -15,15 +15,17 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-var gatewayPort = location.port -9443 + 8243; //Calculate the port offset based gateway port.
-var serverUrl = "https://"+location.hostname +":"+ gatewayPort+"/LogAnalyzerRestApi/1.0";
+var gatewayPort = location.port - 9443 + 8243; //Calculate the port offset based gateway port.
```

Comment:
Hi Tharik,

This is fixed by the commits in : https://github.com/wso2/shared-analytics/pull/59

Thanks,
SajithD